### PR TITLE
Feature/pt 195

### PR DESCRIPTION
**Pull request by luktar**
AuthBottomPanel
image

Auth Footer
image

Helper texts in register forms:
image

### CODE REVIEW

File: src/components/auth/Login.jsx [hunk 1]
```diff
@@ -22,64 +30,92 @@ const Login = () => {
...
+                InputProps={{
+                  startAdornment: (
+                    <InputAdornment position="start">
+                      <Tooltip title={t('register.passwordPrompt')}>
+                        <VpnKeyIcon />
+                      </Tooltip>
+                    </InputAdornment>
+                  ),
+                }}
+                helperText={errors.password && touched.password && errors.password}
```
Comment (AdamSarach) [inline src/components/auth/Login.jsx]: 2x errors.password

File: src/utilities/validationSchema.js [hunk 2]
```diff
@@ -8,3 +8,8 @@ export const RegisterSchema = object().shape({
     .oneOf([ref('password1')], 'Wprowadzone hasła nie są takie same')
     .required('Powtórz hasło'),
 });
+
+export const LoginSchema = object().shape({
+  email: string().email('Email nieprawidłowy').required('Wpisz adres email'),
```
Comment (AdamSarach) [inline src/utilities/validationSchema.js]: translations?

File: src/stylesheets/register.scss [hunk 3]
```diff
@@ -59,5 +47,34 @@ a:hover {
   display:inline-flex;
 }
 
+.register {
```
Comment (AdamSarach) [inline src/stylesheets/register.scss]: I dont understand why you changed flex for grid here

File: src/redux/actions/authActions.js [hunk 4]
```diff
@@ -31,7 +31,6 @@ export const validateLogin = createAsyncThunk('auth/validateLogin', async (data,
 
       throw thunkAPI.rejectWithValue({
         status: response.status,
-        // message: response.error.message,
```
Comment (AdamSarach) [inline src/redux/actions/authActions.js:34]: should be deleted or uncommented?

File: src/components/auth/Register.jsx [hunk 5]
```diff
@@ -37,147 +39,150 @@ const Register = () => {
...
+                    <InputAdornment position="start">
+                      <Tooltip title={t('register.passwordPrompt')}>
+                        <VpnKeyIcon />
+                      </Tooltip>
+                    </InputAdornment>
+                  ),
                 }}
+                helperText={errors.password2 && touched.password2 && errors.password2}
+              />
+              <FormControlLabel
```
Comment (AdamSarach) [inline src/components/auth/Register.jsx:158]: what did you gain by using this component?

File: package.json [hunk 6]
```diff
@@ -69,6 +69,7 @@
   },
   "devDependencies": {
     "@types/leaflet": "^1.5.19",
+    "@types/react-router-dom": "^5.3.0",
```
Comment (AdamSarach) [inline package.json]: why do we need types for react-router-dom?

**Review by AdamSarach — APPROVED** [review]
Im a bit worried about extra work you have done which could be avoided

File: src/components/auth/Register.jsx (hunk 5 above)
Comment (luktar) [reply src/components/auth/Register.jsx:158]: 1. Checkbox is in one line with the label
2. When you click on the text, the checkbox will be checked/unchecked
image

File: package.json (hunk 6 above)
Comment (luktar) [reply package.json]: `npm run start` stopped working with the problem that `react-router-dom` is not available, so every place with `NavLink` prevent me to build the application.
I found this solution somewhere on stack overflow.

File: src/redux/actions/authActions.js (hunk 4 above)
Comment (luktar) [reply src/redux/actions/authActions.js:34]: It was commented before so I removed it.

File: src/components/auth/Login.jsx (hunk 1 above)
Comment (luktar) [reply src/components/auth/Login.jsx]: good point

File: src/redux/actions/authActions.js (hunk 4 above)
Comment (luktar) [reply src/redux/actions/authActions.js:34]: Besides it was duplicated. There was another `message` below.

File: src/stylesheets/register.scss (hunk 3 above)
Comment (luktar) [reply src/stylesheets/register.scss]: There wasn't flex. There was nothing. I've just added a grid to fix the problem on the image.
image

File: src/utilities/validationSchema.js (hunk 2 above)
Comment (luktar) [reply src/utilities/validationSchema.js]: added

**Pull request merged**